modules/cluster/variables.tf

```diff
@@ -436,8 +436,8 @@ variable "use_asm" {
   default     = false
 }
 
-variable "asm_role" {
```

Comment:
keep it here if we are keeping it in variables.tf?

Reply:
Why?

Reply:
nvm, looking at this, makes sense now.

```
boot_iam_role                         = "${var.asm_role}${var.boot_iam_role}"
```